packages/base/adapters/solana/web3js/utils/createSendTransaction.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  PublicKey,
+  SystemProgram,
+  type Connection,
+  Transaction,
+  LAMPORTS_PER_SOL,
+  ComputeBudgetProgram
+} from '@solana/web3.js'
+import type { Provider } from '@web3modal/scaffold-utils/solana'
+
+type SendTransactionArgs = {
+  provider: Provider
+  connection: Connection
+  to: string
+  value: number
+}
+
+export async function createSendTransaction({
+  provider,
+  to,
+  value,
+  connection
+}: SendTransactionArgs): Promise<Transaction> {
+  if (!provider.publicKey) {
+    throw Error('No public key found')
+  }
+
+  const toPubkey = new PublicKey(to)
+  const lamports = Math.floor(value * LAMPORTS_PER_SOL)
+
+  const { blockhash, lastValidBlockHeight } = await connection.getLatestBlockhash()
+
+  const instructions = [
+    ComputeBudgetProgram.setComputeUnitPrice({ microLamports: 20000000 }),
+    ComputeBudgetProgram.setComputeUnitLimit({ units: 500 }),
```

Comment:
are these hardcoded values ok? why specifically this? 
Maybe better to move them to some constants so we know what they mean

Reply:
https://github.com/WalletConnect/web3modal/pull/2802/commits/2265f60d56d6a99c2a5182313911044be85b2d3e